ssl/statem/statem_srvr.c

```diff
@@ -61,6 +61,10 @@ static int ossl_statem_server13_read_transition(SSL *s, int mt)
             break;
         } else if (s->ext.early_data == SSL_EARLY_DATA_ACCEPTED) {
             if (mt == SSL3_MT_END_OF_EARLY_DATA) {
+#ifndef OPENSSL_NO_QUIC
+                if (s->quic_method != NULL)
+                    return 0;
+#endif
```

Comment:
It looks like this does not work very well with 0RTT.  I think we have to fall through to the next case when quic is enabled; I mean `!SSL_IS_QUIC(s) `should be added to line 62.

Reply:
commit 663b4dd8a1 has this change

Reply:
Commit 663b4dd8a1 has this change